lib/Ray/include/Drawables/Drawable.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** Drawable
+*/
+
+#ifndef DRAWABLE_HPP_
+#define DRAWABLE_HPP_
+
+#include <raylib.h>
+#include <Vector.hpp>
+#include "Drawables/Basic/Circle.hpp"
+#include "Drawables/Basic/Line.hpp"
+#include "Drawables/Basic/Rectangle.hpp"
+#include "Drawables/Basic/Point.hpp"
+
+namespace Ray {
+    class Drawable
+	{
+        public:
+            Drawable(Vector2 position, Color color);
+            Drawable(int x, int y, Color color);
+            virtual ~Drawable() = 0;
+
+            const Vector2 &getPosition(void) const;
+            const Color &getColor(void) const;
+
+            Drawable &setPosition(const Vector2 &position);
+            Drawable &setPosition(int x, int y);
+            Drawable &setColor(const Color &color) const;
+
```

Comment:
Maybe add a draw method to simply allow new drawables to be created.

Reply:
'Drawable 'is an abstract class, it shouldn't be instanciated

Reply:
Abstract fileNames should start by 'A' here "ADrawable.hpp"